SpatialGDK/Source/SpatialGDKEditorToolbar/Private/SpatialGDKEditorToolbar.cpp

```diff
@@ -902,7 +902,11 @@ bool FSpatialGDKEditorToolbarModule::StartCloudSpatialDeploymentIsVisible() cons
 
 bool FSpatialGDKEditorToolbarModule::StartCloudSpatialDeploymentCanExecute() const
 {
+#if PLATFORM_MAC
```

Comment:
Perhaps add a comment why it's disabled?

Reply:
I've linked the ticket to make it work.. if so desired.. of course maybe it will be decided not to support this in which case the todo will need to be removed.